scripts/commands/update-translations.js

```diff
@@ -0,0 +1,84 @@
+import {exec} from 'child_process';
+import {promisify} from 'util';
+import https from 'https';
+import fs from 'fs';
+import {error, success, info, gitFactory} from '../utils.js';
+
+const execPromise = promisify(exec);
+const REQUIRED_TRANSLATION_PERCENTAGE = 85;
+
+async function downloadTranslations(url, outputPath) {
+	return new Promise((resolve, reject) => {
+		const file = fs.createWriteStream(outputPath);
+
+		https.get(url, (response) => {
+			response.pipe(file);
+			file.on('finish', () => {
+				file.close();
+				resolve();
+			});
+		}).on('error', (err) => {
+			reject(err);
+		});
+	});
+}
+
+async function runScript() {
+	const git = gitFactory();
+
+	try {
+		info('Executing wp i18n make-pot command...');
+		await execPromise('wp i18n make-pot . languages/wc-calypso-bridge.pot --ignore-domain');
+		success('wp i18n make-pot command executed successfully.');
```

Comment:
How do we know that the `wp i18n make-pot` command executed successfully? Maybe we should check for the existence of the language file?

Reply:
Good idea!